### DISCUSSION TITLE
WIXv4 Burn bootstrapper splash screen launches after Cisco AnyConnect MSI is installed

### DISCUSSION THREAD

**Discussion by Frittions** [wixtoolset/issues#7906] [Questions]
I have a Burn Bootstrapper application that is installing a Cisco AnyConnect msi. I've distilled the install chain down to this single Cisco AnyConnect MSI in the code snippet below for demonstration purposes, but in reality this msi package is one of many in the install chain. 

![image](https://github.com/wixtoolset/issues/assets/97322291/95f41764-2ce8-454b-886d-0797dd4e864c)

The issue is that when bootstrapper installs the Cisco AnyConnect msi, the original bootstrapper will show the success page, as you would suspect, but an additional bootstrapper.exe is launched and shows the splash screen. This is incredibly confusing for the user because they now have a window showing the successful install and a new installation splash screen.

If you look in the task manager you will see that the additional bootstrapper application was called with the following arguments:

![image](https://github.com/wixtoolset/issues/assets/97322291/04c7a6f7-5813-4bc5-9733-8713404fed46)

The main difference I see being that the new process was created with the 'SYSTEM' as the UserName and has been created with unique flags (ie. /burn.clean.room, etc.)

Note: It appears to only have this issue with the Cisco AnyConnect msi file. If I use a different 3rd party msi, I don't get the additional splash screen.

**Comment by robmen** [comment]
Verbose log files are more useful than screenshots. You'll want to start there.
